doc/spelling.md

```diff
@@ -54,8 +54,8 @@ The `path` part is relative to the value at `value-name`. See [#1399](https://gi
 # “foo” dataset at http://api.noms.io/-/aa
```

Comment:
I reckon we should update all of these URLs to be real things at demo.noms.io.

Except bonk I guess.